gitea/Dockerfile

```diff
@@ -0,0 +1,77 @@
+#============================#
+#  ALEXBELGIUM'S DOCKERFILE  #
+#============================#
+#           _.------.
+#       _.-`    ('>.-`"""-.
+# '.--'`       _'`   _ .--.)
+#    -'         '-.-';`   `
+#    ' -      _.'  ``'--.
+#        '---`    .-'""`
+#               /`
+#=== Home Assistant Addon ===#
+
+#################
+# 1 Build Image #
+#################
+
+ARG BUILD_FROM=gitea/gitea
+
+FROM ${BUILD_FROM}
+
+##################
+# 2 Modify Image #
+##################
+
+ENV USER git
```

Comment:
beware that by using a custom USER to run the addon, it could prevent access to the folders that can be used for data remanence and to install the scripts in section 3. of the Dockerfile ; a way around that is to run the addon as root, but then start the app as your specific user

Reply:
thanks! this is a problem that I was trying to solve now! :)

Reply:
I've spent a lot of time also on apps that only accept non-root users :-)

Reply:
Sorry, what are the command to access the folders (i'm trying to access `/share`)?
I removed the `ENV USER git`, but i still can't save things in `/share`

Reply:
To access /share, you neep to add is a map in config.json (I think you did that), but it doesn't get mapped during addon build as this occurs either on github or on a temporary folder. The usual way is either to change the app data folder through an ENV variable (if it exists), or to have a script at boot time that creates the relevant folder than symlinks the folders. Here is an example : https://github.com/alexbelgium/hassio-addons/blob/master/scrutiny/rootfs/etc/cont-init.d/01-run.sh#L8:15